internal/services/projects.go

```diff
@@ -87,6 +87,12 @@ func createProject(
 	var projModel = wrappers.Project{}
 	projModel.Name = projectName
 	projModel.ApplicationIds = applicationID
+	if projectGroups != "" {
+		_, groupErr := GetGroupMap(groupsWrapper, projectGroups, &projModel, nil, featureFlagsWrapper)
```

Comment:
related to previous comment:
groupMap, groupErr = GetGroupMap(groupsWrapper, projectGroups, nil, featureFlagsWrapper)
projModel.Groups = groupMap

Reply:
fixed in a similar way